ebl-issuance/src/main/resources/standards/eblissuance/messages/eblissuance-v3.0.0-request.json

```diff
@@ -87,70 +87,86 @@
       "UNLocationCode": "NLAMS"
     },
     "invoicePayableAt": {
-      "UNLocationCode": "DKAAR"
+      "UNLocationCode": "NLAMS"
     },
     "partyContactDetails": [
       {
-        "name": "DCSA test person",
-        "email": "no-reply@dcsa.example.org"
+        "name": "Henrik",
+        "phone": "+45 51801234"
+      },
+      {
+        "name": "Henrik",
+        "phone": "+45 51801234"
       }
     ],
     "documentParties": {
       "shipper": {
-        "partyName": "DCSA CTK",
+        "partyName": "IKEA Denmark",
         "partyContactDetails": [
           {
-            "name": "DCSA test person",
-            "email": "no-reply@dcsa.example.org"
+            "name": "Henrik",
+            "phone": "+45 51801234"
+          },
+          {
+            "name": "Henrik",
+            "phone": "+45 51801234"
           }
         ]
       },
       "consignee": {
         "partyName": "CONSIGNEE_LEGAL_NAME_PLACEHOLDER",
         "identifyingCodes": [
           {
-            "codeListProvider": "EPUI",
+            "codeListProvider": "W3C",
             "partyCode": "CONSIGNEE_PARTY_CODE_PLACEHOLDER",
             "codeListName": "CONSIGNEE_CODE_LIST_NAME_PLACEHOLDER"
           }
         ],
         "partyContactDetails": [
           {
-            "name": "DCSA test person",
-            "email": "no-reply@dcsa.example.org"
+            "name": "Henrik",
+            "phone": "+45 51801234"
+          },
+          {
+            "name": "Henrik",
+            "phone": "+45 51801234"
           }
         ]
       },
       "issuingParty": {
         "partyName": "Hyundai",
         "address": {
-          "street": "The street name would be here",
-          "city": "... and here the city",
-          "countryCode": "KR"
+          "street": "Ruijggoordweg",
+          "city": "Amsterdam",
+          "countryCode": "NL"
         },
         "identifyingCodes": [
           {
-            "codeListProvider": "SMDG",
-            "codeListName": "LCL",
-            "partyCode": "HMM"
+            "codeListProvider": "W3C",
+            "partyCode": "MSK",
+            "codeListName": "DID"
           }
```

Comment:
The party code is for MSK (and should probably not use `W3C` + `DID`. It looks like an `SMDG` code instead). However, the `partyName` is for Hyundai (HMM).

I assume the original example is wrong. Probably the issuing party should be made a placeholder that the issuer can provide.